assets/install.sh

```diff
@@ -55,8 +55,8 @@ install_file() {
 
   # make links to binaries
   mkdir -p /opt/chef/embedded/bin/
-  ln -sf `which gem` /opt/chef/embedded/bin/
-  ln -sf `which ruby` /opt/chef/embedded/bin/
+  [[ ! -e /opt/chef/embedded/bin/gem ]] && ln -sf `which gem` /opt/chef/embedded/bin/
+  [[ ! -e /opt/chef/embedded/bin/ruby ]] && ln -sf `which ruby` /opt/chef/embedded/bin/
```

Comment:
If we are going to check if the link exists first, then we don't need the `-f` here.